client/lib/application/src/application-lifecycle/components/Toolbar.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import moment from 'moment';
+
+import Brush from 'common/src/components/pure/Brush.jsx';
+import DateSelector, {STYLE_RIGHT} from 'common/src/components/functional/DateSelector.jsx';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+
+const DATE_FORMAT = 'Do [of] MMM YY';
```

Comment:
Uh didn't catch that one last time. We format dates with this: https://github.com/zalando-stups/yourturn/blob/master/client/lib/common/src/config.js#L4